### ISSUE TITLE
[Bug]: Bad del on shuttle flight computers

### ISSUE THREAD

**Issue by Tsar-Salat** [BeeStation/BeeStation-Hornet#10024]
### Describe the bug

I've noticed several instances where destroying computers causes a runtime or tests to fail. I think this is due to a bad /destroy, ~~which Ive fixed in another pr~~

Culprit 

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/a9ec7abe-f756-40a3-b5e6-2260bce79253)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/3e84b2e6-ff47-4c3c-a8a8-53bd6bd1d504)

### What did you expect?

I expected computers getting deleted to not fail my tests in unrelated PRs...

### Reproducible steps

N/A Its very inconsistent when it throws an error or not, see https://github.com/BeeStation/BeeStation-Hornet/actions/runs/6513728417/job/17693823002

**Comment by Tsar-Salat** [comment]
Yeah no, I did NOT fix this. I thought I did, but the del came back in the very *fix* pr I thought dealt with it.

FFS.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/b509f08e-9a09-40aa-809f-bdd87788dc42)

**Comment by itsmeow** [comment]
I think this started happening after #9469. Something to do with the shuttle template loader.

It might also be related to the docking port hyperspace bs

**Comment by Haliris** [comment]
It's not stale, it keeps popping up from time to time.
